src/Sylius/Bundle/AdminBundle/Resources/views/Order/Show/_resendEmail.html.twig

```diff
@@ -1,6 +1,8 @@
-<div class="ui segment">
-    {% set path = path('sylius_admin_order_resend_confirmation_email', {'id': order.id, '_csrf_token': csrf_token(order.id)}) %}
-    <a href="{{ path }}" class="ui icon labeled tiny fluid button" {{ sylius_test_html_attribute('resend-order-confirmation-email') }}>
-        <i class="send icon"></i> {{ 'sylius.ui.resend_the_order_confirmation_email'|trans }}
-    </a>
-</div>
+{% if order.state == 'new'%}
```

Comment:
I'm not sure if this is the only state in which we should resent the email, what about fulfilled? 🤔